NullableAttributes.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// This was copied from https://github.com/dotnet/coreclr/blob/60f1e6265bd1039f023a82e0643b524d6aaf7845/src/System.Private.CoreLib/shared/System/Diagnostics/CodeAnalysis/NullableAttributes.cs
+// and updated to have the scope of the attributes be internal.
+
+#pragma warning disable CA1019 // Define accessors for attribute arguments
+
+namespace System.Diagnostics.CodeAnalysis;
+
+#if NETFRAMEWORK || WINDOWS_UWP || NETSTANDARD && !NETSTANDARD2_1 || NETCOREAPP && !NETCOREAPP3_0_OR_GREATER
```

Comment:
```suggestion
#if NETFRAMEWORK || WINDOWS_UWP || ( NETSTANDARD && !NETSTANDARD2_1 ) || ( NETCOREAPP && !NETCOREAPP3_0_OR_GREATER )
```

Reply:
Not sure if that even works, but it reads better.

Reply:
Will test and fix this in another PR.